sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -662,18 +662,35 @@ object AlluxioUtils extends Logging with Arm {
   // If reading large s3 files on a cluster with slower disks, skip using Alluxio.
   def shouldReadDirectlyFromS3(rapidsConf: RapidsConf, pds: Seq[PartitionDirectory]): Boolean = {
     if (!rapidsConf.enableAlluxioSlowDisk) {
-      return false
-    }
+      logInfo(s"Skip reading directly from S3 because spark.rapids.alluxio.slow.disk is disabled")
+      false
+    } else {
+      var files = pds.flatMap(pd => pd.files).filter { file =>
+        // skip directory
+        !file.isDirectory
+      }
 
-    val files = pds.flatMap(pd => pd.files).filter { file =>
-      !file.isDirectory
-    }
-    val totalSize = files.map { f =>
-      f.getLen
-    }.sum
+      files = files.filter { f =>
+        // only care about parquet and orc files simply, there are other types of files such as
+        // Delta type.
+        f.getPath.getName.endsWith(".parquet") || f.getPath.getName.endsWith(".orc")
```

Comment:
this only works because we only implement it for parquet right now. This would break if we supported like json files.  Otherwise we would first have to check if its delta files before making this assumption.  Can we please separate this out into its own function and then put a big comment there about this.

Reply:
Added comments.
But it's hard to extract a method.
See the build error on Databricks 312:
```
[2022-11-23T03:39:44.559Z]  found   : org.apache.hadoop.fs.FileStatus => Boolean
[2022-11-23T03:39:44.559Z]  required: org.apache.spark.sql.execution.datasources.SerializableFileStatus => Boolean
```

Reply:
ok thats fine then we can deal with it later if needed